front/app/containers/ProjectsShowPage/timeline/PhaseDescription.tsx

```diff
@@ -34,53 +27,54 @@ const Container = styled.div`
   `}
 `;
 
-const Header = styled.div<{ hasContent: boolean }>`
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  margin-bottom: ${(props) => (props.hasContent ? '30px' : '0px')};
-
-  ${isRtl`
-    flex-direction: row-reverse;
-  `}
-`;
-
```

Comment:
Most of these styles were not doing anything, the relevant ones are moved inside PhaseTitle.